content/articles/hashing-in-data-structure/index.md

```diff
@@ -0,0 +1,299 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /hashing-in-data-structure/
+title: Hashing in data structure
+description: In this article, we will explore how the hash function converts a given key to a smaller number and uses the small number as an index in a table called a hash table.
+author: anubhav-bansal
+date: 2021-06-04T00:00:00-16:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /hashing-in-data-structure/hero.jpg 
+    alt: hero image example image
+---
+### Hashing in Data Structure
+
+In hashing, the idea is to use a hash function that converts a given key to a smaller number and uses the small number as an index in a table called a hash table.
+
+We generate a hash for the input using the hash function and then store the element using the generated hash as the key in the hash table.
+
+![HashTable](./HashIntro.PNG)
+
```

Comment:
Are all the images quoted in this article yours? If not, you need to add the sources

Reply:
Yes, all the images are made by me.

Reply:
Okay. Noted. No need of adding any sources.